runtime-linux.md

```diff
@@ -0,0 +1,3 @@
+## File descriptors
+By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the container process by the runtime.
+The runtime may leak additional file descriptors into the container process to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
```

Comment:
i'm fine with calling it an application. I think we should standardize one term for it through out the specs. `application` sounds fine as user may want to run it inside a VM instead of container for example. Then process/init process is not so approriate.